inorbit_republisher/launch/example.launch

```diff
@@ -1,5 +1,5 @@
 <launch>
-  <node name="inorbit_republisher" pkg="inorbit_republisher" type="republisher.py">
+  <node name="inorbit_republisher" pkg="inorbit_republisher" type="republisher.py" output="screen">
```

Comment:
@leandropineda is this a leftover from your tests?

Reply:
No, I left it on purpose so messages printed to stdout are shown.

Reply:
Mmm I see, the only flag I have about this is how verbose the republisher becomes. I believe this is the reason why no ouput to screen was set.

Reply:
I agree, I'll revert it JIC then